docs/vscode_monaco_upgrade.md

```diff
@@ -17,8 +17,11 @@
 
 ## monaco-vscode-api repository
 
-- Update `monaco-editor` and other dependencies and update to the new `vscodeRef` from above
+- Update the package.json to set `monaco-editor` to the same version as the one using the `vscodeRef` from above (usually the latest version)
+- Run `npm install` which will trigger the vscode install script
 - Wait for the new vscode version to be downloaded and built
+- Run `npm run update-vscode-dependencies` to update the vscode dependencies we use to the same versions as vscode
+- Run `npm install` (might need to update/add the npm override for `xterm` in the `package.json` to fix the invalid peer deps)
```

Comment:
can't we make it automatic as well?

Reply:
We can, but I'm not sure it's a good idea. If the override is not necessary on some version I think it'd be better to remove it instead of adding it for nothing ?

Reply:
I just have the feeling they'll always use beta versions and we'll need at each version to keep it and update it by hands

Reply:
Ok for me then: [feat: automatically update overrides for xterm](https://github.com/CodinGame/monaco-vscode-api/pull/210/commits/9251f1bf8bc2ead11c06d5a63467bcbdb43eb330)